crypto/transactions/builder/base.py

```diff
@@ -64,6 +65,19 @@ def multi_sign(self, passphrase, index):
         index_formatted = hex(index).replace('x', '')
         self.transaction.signatures.append(index_formatted + signature.decode())
 
+    def serialize(self, skip_signature=True, skip_second_signature=True, skip_multi_signature=True):
```

Comment:
I'd consider setting them to `False` by default as this is aimed to be used for serialising the transaction prior to sending which requires the signature(s) to be included as well